api/core/evaluate.js

```diff
@@ -128,7 +128,8 @@ export const execute = async (
                 completed.add(id);
                 console.log(`Completed ${id}`);
               } catch (error) {
-                throw error;
+                // This should be reported via a note.
```

Comment:
What are notes?

Reply:
Ah, notes are the elements that notebooks are made of. :)